docs/help-center/docs/warden-app/earn-rewards.md

```
@@ -10,11 +10,7 @@ In this section, you'll discover how to start **earning rewards** in Warden:
 
 1. [Create a coin](#create-your-coin): Create your PumpKoin.
 2. [Earn PUMP points](#earn-pumps): Stay active in Warden to boost your coin's rank.
-3. [Earn WARD](#earn-ward): The top PUMP holders will earn rewards in the WARD token.
-
-:::tip
-The most active users will also gain priority access to upcoming features.
-:::
+3. [Earn rewards](#earn-ward): The top PUMP holders will earn rewards.
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)